File: app/src/main/java/com/boringdroid/settings/BoringdroidSettingsApplication.java
package com.boringdroid.settings;

import android.app.Application;

import androidx.appcompat.app.AppCompatDelegate;

public class BoringdroidSettingsApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.getDefaultNightMode());
    }
}

File: app/src/main/java/com/boringdroid/settings/BoringdroidSettingsApplication.kt
package com.boringdroid.settings

import android.app.Application
import androidx.appcompat.app.AppCompatDelegate

class BoringdroidSettingsApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.getDefaultNightMode())
    }
}
